hdfs/client.py

```diff
@@ -277,6 +278,21 @@ def status(self, hdfs_path, strict=True):
     res = self._get_file_status(hdfs_path, strict=strict)
     return res.json()['FileStatus'] if res else None
 
+  def get_acl(self, hdfs_path, strict=True):
+    """Get AclStatus_ for a file or folder on HDFS.
+
+    :param hdfs_path: Remote path.
+    :param strict: If `False`, return `None` rather than raise an exception if
+      the path doesn't exist.
+
+    .. _AclStatus: ACLS_
+    .. _ACLS: https://hadoop.apache.org/docs/stable2/hadoop-project-dist/hadoop-hdfs/WebHDFS.html#Get_ACL_Status
+
+    """
+    _logger.info('Fetching ACL status for %r.', hdfs_path)
+    res = self._get_acl_status(hdfs_path, strict=strict)
+    return res.json()['GETACLSTATUS'] if res else None
```

Comment:
Should the key be `'AclStatus'`? (From looking at the [WebHDFS doc](https://hadoop.apache.org/docs/stable2/hadoop-project-dist/hadoop-hdfs/WebHDFS.html#Get_ACL_Status).)